src/graph.rs

```diff
@@ -521,11 +541,14 @@ where
     /// in-between (i.e., not an end point), increment their betweenness value.
     /// Normalize the counts by dividing by the number of shortest paths found
     ///
-    pub fn compute_betweenness_and_closeness(&self, agraph: &AGraph) -> (Vec<f64>, Vec<f64>) {
-        let num_nodes = agraph.len();
+    pub fn betweenness_and_closeness_centrality(&mut self) {
```

Comment:
Perhaps this should be part of the private API?

Reply:
done